lib/core/src/sdk.rs

```diff
@@ -927,23 +932,23 @@ impl LiquidSdk {
 
         let receiver_amount_sat = req.prepare_res.receiver_amount_sat;
         let pair = self.validate_chain_pairs(Direction::Outgoing, receiver_amount_sat)?;
-        let claim_fees_sat = pair.fees.claim_estimate();
+        let claim_fees_sat = req.prepare_res.claim_fees_sat;
```

Comment:
How will Boltz become aware of this custom BTC claim feerate? AFAIK they rely on the `pair_hash` arg in the `create_chain_swap` call below to calculate the claim fee.

Reply:
Boltz are using the pair hash and the current fee rates calculate a claim fee estimate. This is only an estimate for the user, as the user broadcasts the claim. So when creating the swap we are setting the serverLockAmount to `amount + custom claim fee`, then when creating the claim tx we can use any fee to broadcast the claim. We use the `custom claim fee`, leaving the `amount` as output